nipype/interfaces/fsl/preprocess.py

```diff
@@ -561,13 +562,18 @@ def _parse_inputs(self, skip=None):
         if isdefined(self.inputs.save_log) and self.inputs.save_log:
             if not isdefined(self.inputs.verbose) or self.inputs.verbose == 0:
                 self.inputs.verbose = 1
+        if isdefined(self.inputs.apply_xfm) and self.inputs.apply_xfm:
+            if not self.inputs.in_matrix_file and not self.inputs.uses_qform:
```

Comment:
shouldn't this be `if not isdefined(self.inputs.in_matrix_file) and not (isdefined(self.inputs.uses_qform)  and not self.inputs.uses_qform)`